app/helpers/keywords_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module KeywordsHelper
```

Comment:
How about removing this auto-generated file also as we are not using it? If needed we can add it later.

Reply:
Good catch! Updated in [`067c022` (#37)](https://github.com/suho/google-search-ruby/pull/37/commits/067c0225bed0a50f9947e8747b64a6b210c6e73d)